app/sagas/selectServer.js

```diff
@@ -11,11 +11,13 @@ import database from '../lib/realm';
 import log from '../utils/log';
 import I18n from '../i18n';
 
-const getServerInfo = function* getServerInfo({ server }) {
+const getServerInfo = function* getServerInfo({ server, raiseError = true }) {
 	try {
 		const serverInfo = yield RocketChat.getServerInfo(server);
 		if (!serverInfo.success) {
-			Alert.alert(I18n.t('Oops'), I18n.t(serverInfo.message, serverInfo.messageOptions));
+			if (raiseError) {
```

Comment:
why did you add `raiseError` parameter?
I suppose that checking `if (!serverInfo.success) {` would be enough to show error Alert.
And besides that `getServerInfo` is called only once in `handleSelectServer`: `getServerInfo({ server, raiseError: false });` so `raiseError` will always be falsy...

Reply:
`getServerInfo` is called on `handleSelectServer`, `handleServerRequest` and `deepLinking`.
I don't want to raise the error every time.